bcipy/main.py

```diff
@@ -99,7 +99,12 @@ def bci_main(
 
     if execute_task(task, parameters, save_folder, alert, fake):
         if visualize:
-            visualize_session_data(save_folder, parameters)
+
+            # Visualize session data and fail silently if it errors
+            try:
+                visualize_session_data(save_folder, parameters)
+            except Exception as e:
+                log.info(f'Error visualizing session data: {e}')
```

Comment:
I'm fine with this approach. I don't think we need to do any further checking for labels.